hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -283,6 +284,13 @@ private FlinkOptions() {
       .withDescription("Payload class used. Override this, if you like to roll your own merge logic, when upserting/inserting.\n"
           + "This will render any value set for the option in-effective");
 
+  public static final ConfigOption<String> MERGE_CLASS_NAME = ConfigOptions
+      .key("write.merge.class")
+      .stringType()
```

Comment:
Should name it `merge.class` if the class is used for reading path, say the MOR table logs.

Reply:
i see, i will change it in step 3. nice advice